components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationMgtDBQueries.java

```diff
@@ -64,6 +64,11 @@ public class ApplicationMgtDBQueries {
             " VALUES (?,?,?,?,?)";
     public static final String STORE_CLAIM_MAPPING = "INSERT INTO SP_CLAIM_MAPPING (TENANT_ID, IDP_CLAIM, SP_CLAIM, " +
             "APP_ID, IS_REQUESTED, IS_MANDATORY, DEFAULT_VALUE) VALUES (?,?,?,?,?,?,?)";
+
+    public static final String UPDATE_SP_IDP_ATTR = "INSERT INTO SP_IDP_ATTR (APP_ID, IDP_ID, ATTR_KEY, " +
+            "ATTR_VALUE) VALUES (?,?,?,?)";
+    public static final String DELETE_SP_IDP_ATTR = "DELETE FROM SP_IDP_ATTR WHERE APP_ID = ? AND ATTR_KEY = ?";
```

Comment:
Please add a new line

Reply:
Addressed in 3da9288685f309f6395405363622af44c39f34c2